apps/catalogue/app.vue

```diff
@@ -1,7 +1,10 @@
 <script setup lang="ts">
 const config = useRuntimeConfig();
 const route = useRoute();
-const { initialize } = useGtag()
+const { initialize } = useGtag();
+
+const datasetStore = useDatasetStore();
+await datasetStore.isDatastoreEnabled();
```

Comment:
I placed this here so that the status is set at the topmost level. I'm not sure if this is the best place to put this or if I should also use `callOnce` here.

Reply:
dunno either but lets keep it here for now.